src/App.js

```diff
@@ -8,7 +8,7 @@ import { Game } from './components/Game/Game';
 
 function App() {
     return (
-        <BrowserRouter>
+        <BrowserRouter basename="/scrabble-workout">
```

Comment:
just a minor comment that shouldn't worry us too much for now
ideally an app code should not care about how it's deployed (to root folder vs to subfolder)

here you assume it's `scrabble-workout` subfolder and specify `basename` prop (it doesn't work properly without it, does it?)

in the future, if we want to deploy somewhere else too (e.g. to scrabble-workout.com) then we will have to update this, like https://create-react-app.dev/docs/deployment#customizing-environment-variables-for-arbitrary-build-environments